samples/iotc-sensors-gps/src/main.c

```diff
@@ -332,68 +529,46 @@ static void memory_test() {
 static int sdk_run() {
     int err;
     sdk_running = true;
-    ui_led_set_rgb(LED_MAX, LED_MAX, 0);
+    time_t now = 0;
```

Comment:
I am not a big fan off declaring variables at the top of the function before they are used. They were all declared in the section below in order to limit their scope. I will not argue with this change, but I hope we keep this phillosophy in the future.

Reply:
noted.  copied those changes from iot-basic sample to make them consistent.